llvm/lib/Target/X86/MCTargetDesc/X86MCCodeEmitter.cpp

```diff
@@ -121,11 +126,14 @@ class X86OpcodePrefixHelper {
   unsigned R : 1;
   unsigned X : 1;
   unsigned B : 1;
+  unsigned M : 1;
```

Comment:
No representation in EVEX encoding?

Reply:
Yes. `M` bit is used encode MAP 0/1 in REX2. In EVEX, we use `mmmm`